lib/kernel/svm_atomics_x86_64.ll

```diff
@@ -1,8072 +0,0 @@
-; ModuleID = 'svm_atomics_host.cl.bc'
```

Comment:
Why this file is deleted?

Reply:
There were multiple versions of "atomics" file for various older LLVM versions, because of changes around atomics in bitcode or builtins, i can't remember now. Those LLVM versions can be seen in lines removed from `lib/kernel/host/CMakeLists.txt` in same commit.